src/pages/storage/forms/StoragePoolFormZFS.tsx

```diff
@@ -19,7 +24,20 @@ const StoragePoolFormZFS: FC<Props> = ({ formik }) => {
           label: "ZFS pool name",
           name: "zfs_pool_name",
           defaultValue: "",
-          children: <Input type="text" placeholder="Enter ZFS pool name" />,
+          children: !isClusteredServer(settings) ? (
+            <Input type="text" placeholder="Enter ZFS pool name" />
+          ) : (
+            <ClusteredZfsNameSelector
+              formik={formik}
+              placeholder="Enter ZFS pool name"
+            />
+          ),
```

Comment:
Better to keep the conditional without negation. That tends to be much easier to read. Especially when you have also an else branch it is preferred to swap them and keep the conditional positive.